packages/smooth_app/lib/cards/product_cards/knowledge_panels/knowledge_panel_title_card.dart

```diff
@@ -20,12 +20,14 @@ class KnowledgePanelTitleCard extends StatelessWidget {
     Color? colorFromEvaluation;
```

Comment:
`late Color colorFromEvaluation` would make more sense.

Reply:
I @monsieurtanuki , I tried this but it raises LateInitializationError and therefore I haven't changed it.